lms/util/lti.py

```diff
@@ -0,0 +1,30 @@
+from pyramid.httpexceptions import HTTPBadRequest
+
+from lms import models
+
+
+def lti_params_for(request):
+    """
+    Return the LTI params for the given request.
+
+    If the request is an LTI launch request then just return request.params.
```

Comment:
I'm guessing what constitutes an "LTI launch request" is the presence of certain params on `request`? Will the reader of this docstring know that?

Edited: This question was answered by the code and comments starting at line 20 https://github.com/hypothesis/lms/pull/284/commits/baaa2ab6a4811a01b750ba285d41fe3a37acd20c#diff-64b3713b087cbdc433f9602eabd447eeR20